cmd/abigen/main.go

```diff
@@ -96,7 +96,7 @@ var (
 	}
 	aliasFlag = cli.StringFlag{
 		Name:  "alias",
-		Usage: "Comma separated aliases for function and event renaming, e.g. foo=bar",
+		Usage: "Comma separated aliases for function and event renaming, e.g. foo=_foo, bar=_bar",
```

Comment:
IMO this doesn't make it much clearer to me. Is `_foo` the alias, or the name? Perhaps `alias=original,foo=_foo` (if that indeed is correct)

Reply:
is it alias=original? Thought it was original=alias

Reply:
Yeah I totally have no idea :)  Let's use whichever is correct, and make it so the usage-string makes it clear which it is

Reply:
just tried it out again to confirm - it's original=alias - so my suggestion
```suggestion
		Usage: "Comma separated aliases for function and event renaming, e.g. original1=alias1, original2=alias2",
```

Reply:
Should we just make the changes by ourselves and merge? It's a trivial PR.

@huahuayu care to make the change?